src/tests/topp/DatabaseSuitability_out_1.tsv

```diff
@@ -1,7 +1,7 @@
 key	value
-#top_db_hits	158

```

Comment:
This change was expected?

Reply:
Yeah, it was. Since the default for the `rerank_cutoff_percentile` changed, I had to change either the test or the test files.
I decided to change the test files, so that the default is tested.

Reply:
Ah okay